Makefile

```diff
@@ -0,0 +1,22 @@
+##
+# Valtan
+#
+
+LISP ?= sbcl
+INSTALLDIR=/usr/bin
+PROGNAME=valtan
+
+deps:
+	git submodule update --init --recursive
+
+build: deps
+	$(LISP) --eval '(asdf:make :valtan-cli/executable)' \
+		--eval '(quit)'
+
+install:
+	cp $(PROGNAME) $(INSTALLDIR)/$(PROGNAME)
+
+uninstall:
+	rm $(INSTALLDIR)/$(PROGNAME)
```

Comment:
These two may not be my favorite.
The user may have to decide where to install them. (e.g. $HOME/.local/bin, $HOME/bin)